src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts

```diff
@@ -138,7 +139,7 @@ export async function onRemoveEmail(email: EmailRow) {
     return {
       success: false,
       error: "delete-email-without-active-session",
```

Comment:
What is this `error` ID lookin' thing?
I assumed it was an FTL id and `errorMessage` was some fallback text, but not seeing other instances of "delete-email-without-permission" in the codebase.

```sh
[MNTOR-2955] $ git grep -in 'delete-email-without-permission' | cat
src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts:154:      error: "delete-email-without-permission",
```

Also tried looking up by the `errorMessage` but didn't find matches in .ftl files and am wondering if this is possibly just hardcoded English in the dashboard:
```sh
[MNTOR-2955] $ git grep -in -B1 'without an active session' | cat
src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts-136-    logger.error(
src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts:137:      `Tried to delete email [${email.id}] without an active session.`,
--
src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts-141-      error: "delete-email-without-active-session",
src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/actions.ts:142:      errorMessage: `User tried to delete email without an active session.`,
```

Reply:
I don't think we should translate error messages, they aren't shown directly to customers - they are for consumers of the API. We just need some kind of identifier that will show up in our logs.